R/uniDimCommon.R

```diff
@@ -0,0 +1,141 @@
+
+
+.readData <- function(dataset, options) {
+
+  variables <- unlist(options[["variables"]])
+  if (is.null(dataset)) {
+    if (options[["missingValues"]] == "excludeCasesListwise")
+      dataset <- .readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = NULL, exclude.na.listwise = TRUE)
+    else
+      dataset <- .readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = NULL, exclude.na.listwise = NULL)
```

Comment:
I wish `FALSE` was an accepted option but it is not.
```suggestion
    .readDataSetToEnd(
      columns.as.numeric  = variables, 
      exclude.na.listwise = if (options[["missingValues"]] == "excludeCasesListwise") TRUE else NULL
    )
```